app/dev/controllers/capabilitiesController.js

```diff
@@ -14,19 +18,23 @@ app.controller('CapabilitiesController', ['$scope', '$http', '$rootScope', funct
           url: newCapability.capabilityUrl,
           oauth: newCapability.capabilityOauth
         }
-      }).success(function(response) { $scope.capsInCell.push(response) })
-      .error(function() { console.log("error adding capability") });
+      }).success(function(response) {
+        $scope.capsInCell.push(response)
+      }).error(function() {
+        console.log("error adding capability")
+      });
     }
 
-    $scope.updateCapability = function(capability) {
+    $scope.updateCapability = function(capability, id) {
```

Comment:
Is this the capability ID we are passing in? If so, its already part of the capability we are passing in.